pages/linux/login.md

```diff
@@ -0,0 +1,19 @@
+# Login
```

Comment:
```suggestion
# login
```

Reply:
@en1gma713, just this one and should be fine. 👍🏻 We tend to use lowercase titles for the pages. Cheers. 🙂